source/documentation/partials/_messagetypeintro.md

```diff
@@ -1,4 +1,20 @@
-## Accessing the XML Schemas
+## Message details
+
+This section provides the following information about each message type: 
+
```

Comment:
I wonder if this intro needs to be reworded. The problem is "This section...", which seems too redundant or self-referential. I would be inclined to keep it simple.

Suggestion: Changes to message type details are as follows:

Reply:
Done.